crates/red_knot_project/src/db.rs

```diff
@@ -60,7 +60,7 @@ impl ProjectDatabase {
     }
 
     pub fn check_file(&self, file: File) -> Result<Vec<Box<dyn OldDiagnosticTrait>>, Cancelled> {
-        let _span = tracing::debug_span!("check_file", file=%file.path(self)).entered();
+        let _span = tracing::debug_span!("check_file", ?file).entered();
```

Comment:
I created a custom `Debug` implementation that only prints the path unless `{:#?}` is used which seems the better default and simplifies the log calls